src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: default
+title: "Commonly Used Dart Libraries"
+description: "What are some of the most useful and popular Dart libraries and where can you learn more?"
+---
+
+You can leverage a wide array of available Dart libraries when writing your code.
+Dart libraries are accessed in one of three ways:
+
+* The core libraries, such as dart:core, dart:async, and dart:collections,
+  are distributed with the SDK and documented at [api.dartlang.org]({{site.dart_api}}).
+* Libraries shared with the Dart community are distributed as library packages,
+  published at [pub.dartlang.org](https://pub.dartlang.org/), and documented at
+  [dartdocs.org](https://www.dartdocs.org/). The [pub](/tools/pub/) tool
+  allows you to create, publish, and manage library packages.
+* Libraries that are not shared with the community are simply placed in the
+  `/lib` directory of your [application's directory
+  structure](https://www.dartlang.org/tools/pub/package-layout#public-directories).
+
+This document talks about the first two kinds of libraries,
+and tells you where to learn more about some of the most widely used Dart libraries.
+
+<aside class="alert alert-info" markdown="1">
```

Comment:
This seems like a lot of 'boiler plate' before we get to the interesting part, namely: the table. Would it make sense to put these notes below? Or make them much shorter?

This applies to these asides in particular, but also to everything above the table in general. The table is the meat of this article, and for 2 screens-heights we're serving salad.